### PR TITLE
Introduction UA

### PR DESCRIPTION
**Pull request by csagataj2**

### CODE REVIEW

**Comment by qwrtln** [comment]
Please attach screenshots of the translated pages.

**Comment by qwrtln** [comment]
You can use the pdf2image.sh script in the repo

**Review by qwrtln — CHANGES_REQUESTED** [review]
Please fix the frame

File: translations/introduction.tex/ua.po [hunk 1]
```diff
@@ -73,6 +79,30 @@ msgid ""
...
+"\\textit{Курсивом} виділено приклади ігрового процесу.\n"
+"\\hyperlink{Heroes of Might and Magic III}{Гіперпосилання коричневого кольору} ведуть до інших частин книги правил.\n"
+"% The following glyphs MUST be rendered before their first use in tabular environment in all_map_locations.tex.\n"
+"% Inkscape will fail otherwise.\n"
+"% Other glyphs referenced over there ONLY cause no issues ¯\\_(ツ)_/¯\n"
+"\\phantom{\n"
+"  \\includesvg[height=0.1px]{\\svgs/2_treasure_die.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/arrow_right_gray.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/artifact.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/bronze-table.svg}\n"
```
Comment (qwrtln) [inline translations/introduction.tex/ua.po:94]:
```suggestion
```
This one's going to disappear.

**Comment by qwrtln** [comment]
po4a check

**Comment by csagataj2** [comment]
po4a commit

File: translations/introduction.tex/ua.po [hunk 2]
```diff
@@ -72,6 +79,29 @@ msgid ""
...
+"  \\includesvg[height=0.1px]{\\svgs/2_treasure_die.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/arrow_right_gray.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/artifact.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/damage-table.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/movement.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/spellpower.svg}\n"
+"}\n"
+"\\vfill\n"
+"\\columnbreak\n"
+"\\note{11}{Винятки та примітки з \\hyperlink{Heroes of Might and Magic III}{гіперпосиланнями} пояснюються у вікнах подібних до цього.\n"
```
Comment (qwrtln) [inline translations/introduction.tex/ua.po:100]:
```suggestion
"\\note{10}{\\vspace{1em}Винятки та примітки з \\hyperlink{Heroes of Might and Magic III}{гіперпосиланнями} пояснюються у вікнах подібних до цього.\n"
```

Comment (qwrtln) [reply translations/introduction.tex/ua.po:100]: This is how it looks like with the change:
ua-04

Comment (qwrtln) [reply translations/introduction.tex/ua.po:100]: This is how it looks like with the change:

ua-04

File: translations/introduction.tex/ua.po [hunk 3]
```diff
@@ -72,6 +79,29 @@ msgid ""
...
+"  \\includesvg[height=0.1px]{\\svgs/2_treasure_die.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/arrow_right_gray.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/artifact.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/damage-table.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/movement.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/spellpower.svg}\n"
+"}\n"
+"\\vfill\n"
+"\\columnbreak\n"
+"\\note{10}{\\vspace{1em}Винятки та примітки з \\hyperlink{Heroes of Might and Magic III}{гіперпосиланнями} пояснюються у вікнах подібних до цього.\n"
```
Comment (qwrtln) [inline translations/introduction.tex/ua.po:100]:
```suggestion
"\\note{10}{Винятки та примітки з \\hyperlink{Heroes of Might and Magic III}{гіперпосиланнями} пояснюються у вікнах подібних до цього.\n"
```
No longer necessary.

**Review by qwrtln — APPROVED** [review]

**Pull request merged**